api/src/api/userRouter.ts

```diff
@@ -74,6 +79,11 @@ const shouldCheckLicense = (currentBusiness: Business): boolean => {
   return hasLicenseDataOrFormationAddress && licenseDataOlderThanOneHour;
 };
 
+const shouldCheckXrayRegistration = (currentBusiness: Business): boolean =>
+  currentBusiness.xrayRegistrationData !== undefined &&
+  currentBusiness.xrayRegistrationData.lastUpdatedISO !== undefined &&
```

Comment:
Question - Do we need the migration? Let's say we don't do it, and this fn is updated to return true when lastUpdatedISO is undefined. The api call will be made and we can return the results with lastUpdatedISO

Reply:
We will still need to update userData, and maybe have an empty migration file